README.md

```diff
@@ -44,6 +44,17 @@ You can download the pre-built binaries from the [release page](https://github.c
 cargo install tenere
 ```
 
+### ❄️ NixOS / Nix
+
+Tenere is available in nixpkgs and can be installed via configuration.nix:
+
+```nix
+environment.systemPackages = with pkgs; [
+  tenere
+];
+```
+or ```nix-env -iA nixpkgs.tenere``` on Non-NixOS systems
```

Comment:
maybe you can have it in multiple lines, Something like

```
Or as following for Non-NixOs systems
\```
nix-env -iA nixpkgs.tenere
\```
```